### PR TITLE
Changes to run on cyverse

### PR DESCRIPTION
**Pull request by dannymandel**
@datadavev -- in case you're curious, these are the changes to get things up on henry.cyverse.org so far

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "isamples_inabox"]
 	path = isb/isamples_inabox
-	url = git@github.com:isamplesorg/isamples_inabox.git
+	url = https://github.com/isamplesorg/isamples_inabox.git
```
Comment (dannymandel) [inline .gitmodules:3]: Using https works without ssh credentials, hence the change.

File: docker-compose.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ services:
         environment:
             - POSTGRES_PASSWORD=isamplesinabox
             - POSTGRES_USER=isb_writer
-            - POSTGRES_DB=isb_1
+            - POSTGRES_DB=isb_2
```
Comment (dannymandel) [inline docker-compose.yml:8]: The issue I ran into here is that the format of the db dump file I used failed miserably if the db already existed.  So, I created a new one and imported into there.  Long term, I think I need to redo the dump to only include data (not table and index creation), and try and import that way.  But short term while figuring this all out I just recreated the db and brought up the container with the new db name but left iSB pointing at the old one (so it wouldn't recreate).  Another option is to change the iSB app to not create the schema.  One of those things that is a still a bit manual at this point.

File: isb/Dockerfile [hunk 3]
```diff
@@ -3,16 +3,26 @@ FROM python:latest
 
 WORKDIR /app
 
+# The installation of the python requirements should be basically the first thing here.  Docker files will re-run steps
+# that appear after a step that has changed.  Setting up the python environment is the most time-consuming step; put
+# it first so it only executes when it absolutely has to.
 COPY ./isamples_inabox/requirements.txt requirements.txt
 RUN pip3 install -r requirements.txt
 
+RUN apt-get update -y && apt-get install -y libgeos-dev && apt-get install -y cron
```
Comment (dannymandel) [inline isb/Dockerfile:12]: Need this to install dependencies in the container before we try and run the python that needs it.

File: solr/Dockerfile [hunk 4]
```diff
@@ -1,3 +1,3 @@
 # syntax=docker/dockerfile:1
 FROM solr:latest
-COPY ./jts-core-1.15.0.jar /opt/solr-8.9.0/server/solr-webapp/webapp/WEB-INF/lib/
\ No newline at end of file
+COPY ./jts-core-1.15.0.jar /opt/solr-8.10.1/server/solr-webapp/webapp/WEB-INF/lib/
```
Comment (dannymandel) [inline solr/Dockerfile:3]: This is kind of a pain -- I didn't see a way to get this directory out of an environment variable or something.  Until we figure this out, anytime they rev the solr docker image we'll need to update our stuff.  I suppose we could pin our stuff to a particular version of solr, too.  Something to figure out.

**Review by datadavev — APPROVED** [review]

File: docker-compose.yml [hunk 5]
```diff
@@ -7,23 +7,31 @@ services:
...
+            - pg_data:/var/lib/postgresql/data
         
     solr:
         build: ./solr/
         volumes:
-            - solrdata:/var/solr
+            - solr_data:/var/solr
         ports:
-            - "8983:8983"
+            - "8983"
```
Comment (dannymandel) [inline docker-compose.yml:17]: No longer expose the solr port to the world.

**Pull request merged**